### ISSUE TITLE
Chart cannot display profit larger than 1000

### ISSUE THREAD

**Issue by FozzyGo4** [iNextrix/ASTPP#72]
The dashboard chart does not display profit correctly if value is larger than 1000 as Highchart does not handle the thousand separator correctly. Example:

Profit value = 1200.00
With LOCALE en_US thousand separator is ","
This makes the chart value 1,200.00

The chart will show 1 and not 1,200.00 as it breaks the value after the ",".

**Comment by dev-astpp** [comment]
As we check in our system it display correct value for all dashboard graph. So can you please provide screenshot?

**Comment by FozzyGo4** [comment]
Ok see screenshots. I am only displaying the profit line on the graph in this example. I have changed line 133 on /astpp/application/modules/dashboard/controllers/dashboard.php and removed the currency function from profit to make it work correctly. Line 133 is commented out for line 134.

```
		  //$json_data['profit'][]=  array((string)$acc_arr[$day]['day'],(float) $this->common_model->calculate_currency($acc_arr[$day]['profit']));
		  $json_data['profit'][]=  array((string)$acc_arr[$day]['day'],(float) $acc_arr[$day]['profit']);
```

Day 10 is showing 1 ZAR as profit but it should be 1186.34 ZAR. Day 8 and any other day that is bellow 1000 displays correct because there is no , for a thousand separator.
![shot6](https://cloud.githubusercontent.com/assets/20914943/17846227/a17a520e-6847-11e6-93eb-aeda310a82dd.png)

Once I applied the code above to not run the currency function, it displays the graph correctly because there is no , thousand separator.
![shot7](https://cloud.githubusercontent.com/assets/20914943/17846304/0e77b216-6848-11e6-92f2-f49f4785af49.png)
